src/ServiceStack.Mvc/ServiceStackController.cs

```diff
@@ -229,10 +250,6 @@ protected virtual TUserSession SessionAs<TUserSession>()
         {
             return ServiceStackProvider.SessionAs<TUserSession>();
         }
-        protected virtual void SaveSession(IAuthSession session, TimeSpan? expiresIn = null)
-        {
-            ServiceStackProvider.Request.SaveSession(session, expiresIn);
-        }
```

Comment:
Was working of code from a couple of days ago, I'm guessing you added this after